pkg/pb/synthetic_monitoring/checks.proto

```diff
@@ -369,4 +369,5 @@ message TracerouteSettings {
 	int64 maxUnknownHops = 2 [(gogoproto.jsontag) = "maxUnknownHops"]; // Maximum hops probe that give no response before giving up
 	bool  ptrLookup 		 = 3 [(gogoproto.jsontag) = "ptrLookup"];      // Include reverse DNS lookup
 	int64 hopTimeout 		 = 4 [(gogoproto.jsontag) = "hopTimeout"];		 // Timeout for individual hop pings
+	int64 firstHop 		 	 = 5 [(gogoproto.jsontag) = "firstHop"];		   // TTL to start at
```

Comment:
this doesn't seem to be used?

Reply:
you're right, I stubbed it out for now because it's something I'd like to add in the future, but maybe I'll just remove it until it's actually time to add it to prevent confusion down the line